src/EnergyPlus/UnitarySystem.cc

```diff
@@ -9514,7 +9524,7 @@ namespace UnitarySystems {
                                     TempSysOutput = TempSensOutput;
                                 }
                                 TempMinPLR = TempMaxPLR;
-                                while ((TempSysOutput - TempLoad) < 0.0 && TempMinPLR > 0.05) {
+                                while ((TempSysOutput - TempLoad) < 0.0 && TempMinPLR > 0.025) {
```

Comment:
@lymereJ why was this changed? The PLR increment is 0.1, 0.05 splits the difference between 0.0 and 0.1. Did changing this to 0.025 correct something? This doesn't really bother me because 0.025 is still between 0.0 and 0.1. It's more of a curiosity that I ask. Also, this is way into the weeds in UnitarySystem so I have to think you stepped through the debugger for some time looking for issues in order to get here. I am guessing you don't recall why you changed this but am hoping you do recall.

Reply:
Yes, It's been a while but if I recall correctly, I observed that when running the economizer at high speed, sometimes not all the load could be met and that the cooling coil had to run and in some of these instances the load left to be met by the coil was very small and so would be the PLR. Because of that limit at 0.05 I was getting some convergence warning, I believe that lowering the min PLR here helped with that.

Reply:
Oh yeah, I didn't look far enough down the code. The increment on minPLR is 0.01 (I recall that now, 0.1 on max, then reduce to 0.01 on min). So this could (should?) be lowered to 0.005 (I might have mistakenly left a 0 out when I wrote this years ago). Since you have tested this I guess leave if for now and move on. If you have other changes to make please put a comment here that the lower limit might be changed to 0.005 without adverse affects.

                                while ((TempSysOutput - TempLoad) < 0.0 && TempMinPLR > 0.025) {
                                    // pull upper limit of HeatingPLR down to last valid limit (i.e. heat output still exceeds SystemSensibleLoad)
                                    TempMaxPLR = TempMinPLR;
                                    // find minimum limit of HeatingPLR
                                    TempMinPLR -= 0.01;